nautobot/dcim/views.py

```diff
@@ -392,10 +393,24 @@ class RackElevationListView(generic.ObjectListView):
     """
 
     queryset = Rack.objects.prefetch_related("role")
+    non_filter_params = (
+        *generic.ObjectListView.non_filter_params,
+        "face",  # render front or rear of racks?
+        "reverse",  # control of ordering
+    )
 
     def get(self, request):
```

Comment:
Let's make sure this is the only place we are overloading a ListView's `get()`

Reply:
The only other place I see in the code is in `ObjectChangeListView`, and it calls `super().get(...)` so it's fine.